### PR TITLE
Use AllowShutdown for KDM (bnc#907907)

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: src/include/security/levels.rb [hunk 1]
```diff
@@ -115,7 +115,7 @@ def initialize_security_levels(include_target)
             "FAIL_DELAY"                                => "6",
             "GID_MAX"                                   => "60000",
             "GID_MIN"                                   => "1000",
-            "DISPLAYMANAGER_SHUTDOWN"                   => "root",
+            "AllowShutdown"                             => "Root",
```
Comment (jreidinger) [inline src/include/security/levels.rb]: really root with big `R` what it is? it is not user name

File: src/scrconf/kde4_kdmrc.scr [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * File:	kde4_kdmrc.scr
```
Comment (jreidinger) [inline src/scrconf/kde4_kdmrc.scr]: this file missing in Makefile.am

File: src/scrconf/kde4_kdmrc.scr [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * Access:	read / write
+ *
+ * Read/Sets the values defined in
+ * <tt>/usr/share/kde4/config/kdm/kdmrc</tt>
+ */
+.kde4.kdmrc
+
+`ag_ini(
+  `IniAgent(
+    "/usr/share/kde4/config/kdm/kdmrc",
```
Comment (jreidinger) [inline src/scrconf/kde4_kdmrc.scr]: so we really modify configuration in `/usr/share`? it smells for me.

File: src/include/security/levels.rb (hunk 1 above)
Comment (ancorgs) [inline src/include/security/levels.rb]: Yes. From /usr/share/kde4/config/kdm/kdmrc
```
# Who is allowed to shut down the system. This applies both to the
# greeter and to the command sockets.
# "None" - no "Shutdown..." menu entry is shown at all
# "Root" - the root password must be entered to shut down
# "All" - everybody can shut down the machine
# Default is All
```
Actually, I'm quite sure it's case insensitive in the end but according to documentation, it should be "Root".

Comment (jreidinger) [inline src/include/security/levels.rb]: ok, not much intuitive. Looks like all desktop developers enjoy creating another layers of abstraction with own names.

File: src/scrconf/kde4_kdmrc.scr (hunk 2 above)
Comment (ancorgs) [inline src/scrconf/kde4_kdmrc.scr]: Good catch. Fixed.

File: src/scrconf/kde4_kdmrc.scr (hunk 3 above)
Comment (ancorgs) [inline src/scrconf/kde4_kdmrc.scr]: I also found it smelly, but it was stated in the bug report as the proper place. I will ask some KDE packager just to be sure.

Comment (ancorgs) [inline src/scrconf/kde4_kdmrc.scr]: Confirmed by tittiatcoke that it's the expected location.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**